interface/RecoObjects/Jet.h

```diff
@@ -79,9 +82,11 @@ class Jet: public Particle {
 	//Quarks: d=1, u=2, s=3, c=4, b=5 (antimatter with flipped sign)
 	// gluons = 21
 	int partonFlavour() const;
+	double EnergyRaw() const;
 	double PxRaw() const;
 	double PyRaw() const;
 	double PzRaw() const;
+	double getJEC() const;
```

Comment:
This conflicts with the member variable "JEC".  I can rename the member variable to "jec" ?